packages/sdks-tests/src/snippet-tests/live-previewing.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { expect } from '@playwright/test';
+import { test } from '../helpers/index.js';
+import { launchEmbedderAndWaitForSdk, sendPatchOrUpdateMessage } from '../helpers/visual-editor.js';
+
+test.describe('LivePreviewBlogData Component', () => {
+  test('should render the page without 404', async ({ page, packageName }) => {
+    test.skip(!['react'].includes(packageName));
+
+    const response = await page.goto('/live-preview');
+    expect(response?.status()).toBeLessThan(400);
+  });
+
+  test('should display blog details correctly', async ({ page, packageName }) => {
+    test.skip(!['react'].includes(packageName));
+
+    await page.goto('/live-preview');
+    await page.waitForLoadState('networkidle');
```

Comment:
We shouldn't need this check. https://playwright.dev/docs/api/class-page#page-wait-for-load-state

> Most of the time, this method is not needed because Playwright [auto-waits before every action](https://playwright.dev/docs/actionability).

```suggestion
```

Any particular reason you needed it?